File: app/src/main/java/com/innov8/memeit/Adapters/MyViewHolder.java
package com.innov8.memeit.Adapters;

import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by Jv on 7/12/2018.
 */

public abstract class MyViewHolder<T> extends RecyclerView.ViewHolder {


    public MyViewHolder(View itemView) {
        super(itemView);
    }

    public abstract void bind(T t);
}

File: commons/src/main/java/com/innov8/memeit/commons/MyViewHolder.kt
package com.innov8.memeit.commons

import android.view.View
import androidx.recyclerview.widget.RecyclerView.ViewHolder

/**
 * Created by Jv on 7/21/2018.
 */

abstract class MyViewHolder<T>(itemView: View) : ViewHolder(itemView) {

    abstract fun bind(t: T)

}
